### PR TITLE
test: adding some debug logs to diagnose hang

### PR DESCRIPTION
**Pull request by alyssawilk**
Risk Level: n/a (test only)
Testing: yep
Docs Changes: no
Release Notes: no

#6874

### CODE REVIEW

**Comment by alyssawilk** [comment]
I can't repro #6874 locally - this is my best idea is to try to get some more information.  
Feel free to reject as really kludgy :-P

File: test/integration/load_stats_integration_test.cc [hunk 1]
```diff
@@ -454,8 +454,12 @@ TEST_P(LoadStatsIntegrationTest, LocalityWeighted) {
   locality_weighted_lb_ = true;
   initialize();
 
+  // Debug logs for #6874
+  std::cerr << "Waiting for load stats stream.\n";
```
Comment (lizan) [inline test/integration/load_stats_integration_test.cc]: I think you'll need use `std::endl` as end of line so it flushes the buffer.

**Comment by mattklein123** [comment]
Hopefully not needed anymore via https://github.com/envoyproxy/envoy/pull/6784

**Comment by alyssawilk** [comment]
lizan merged commit 27b3e48 into envoyproxy:master  3 days ago

Sat, May 11, 8:07 PM (2 days ago)
https://circleci.com/gh/envoyproxy/envoy/215476?utm_campaign=workflow-failed&utm_medium=email&utm_source=notification
[ RUN      ] IpVersions/LoadStatsIntegrationTest.LocalityWeighted/IPv4

**Comment by alyssawilk** [comment]
Thanks.  One more hang last night
https://circleci.com/gh/envoyproxy/envoy/217508?utm_campaign=workflow-failed&utm_medium=email&utm_source=notification
so hopefully next time we'll get more info!

**Pull request merged**